api/v1beta2/ocimanagedcluster_webhook.go

```diff
@@ -311,7 +311,7 @@ func (c *OCIManagedCluster) GetControlPlaneEndpointDefaultEgressRules() []Egress
 			EgressSecurityRule: EgressSecurityRule{
 				Description:     common.String("Allow Kubernetes API endpoint to communicate with OKE."),
 				Protocol:        common.String("6"),
-				DestinationType: EgressSecurityRuleSourceTypeServiceCidrBlock,
+				DestinationType: EgressSecurityRuleDestinationTypeServiceCidrBlock,
```

Comment:
What his a mistake being fixed?

Reply:
I just renamed the Egress rule enum name to be Destination rather than source, since Egress rules are destination based. The other egress constants were called as Destinations, this one was named incorrectly.  The enum value is not being changed though